games-action/polymc/polymc-1.4.1-r2.ebuild

```diff
@@ -85,6 +85,10 @@ RDEPEND="
 	virtual/opengl
 "
 
+PATCHES=(
+	"${FILESDIR}"/${PN}-explicit_include_in_PollServer.patch
```

Comment:
The name of the patch should include the version number it was first introduced in, i.e. `PN` --> `P`